{{cookiecutter.snake_case_slug}}/README.md

```diff
@@ -18,3 +18,31 @@ $ npm start
 ```
 
 The terminal should automatically open a new browser window and navigate to `localhost:3000`. You can also do this manually.
+
+### Included libraries based on options
+- spring_boot_authentication
```

Comment:
Shouldn't this be a cookiecutter conditional, i.e. `{% if spring_boot_authentication == "y"}...{% endif %}`?